hazelcast/src/main/java/com/hazelcast/internal/nearcache/impl/invalidation/RepairingHandler.java

```diff
@@ -29,33 +29,32 @@
 
 /**
  * Handler used on Near Cache side. Observes local and remote invalidations and registers relevant
- * data to {@link MetaDataContainer}s
+ * data to {@link MetaDataContainer}s.
  *
- * Used to repair Near Cache in the event of invalidation-event-miss or partition uuid changes.
+ * Used to repair Near Cache in the event of missed invalidation events or partition uuid changes.
  * Here repairing is done by making relevant Near Cache data unreachable.
  * To make stale data unreachable {@link StaleReadDetectorImpl} is used.
  *
  * An instance of this class is created per Near Cache and can concurrently be used by many threads.
  *
  * @see StaleReadDetectorImpl
  */
-@SuppressWarnings({"checkstyle:nestedifdepth", "checkstyle:npathcomplexity"})
 public final class RepairingHandler {
 
-    private final int partitionCount;
-    private final String name;
+    private final ILogger logger;
     private final String localUuid;
+    private final String name;
     private final NearCache nearCache;
-    private final ILogger logger;
     private final MinimalPartitionService partitionService;
```

Comment:
In the next PR the `SerializationService` will be added here, therefore some re-ordering to group services and their usage etc.